layers/+lang/typescript/packages.el

```diff
@@ -89,7 +91,11 @@
                           (progn
                             (with-current-buffer outputbuf (erase-buffer))
                             (write-region nil nil tmpfile)
-                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil (list tmpfile)))
+                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil
+                                              (list (format
+                                                     "--baseDir='%s' --"
+                                                     default-directory)
```

Comment:
I'm not sure that default-directory is the best choice. E.g when one start docker at boot time, or use the same instance to edit files in different projects.